test/modules/CloudToDeviceMessageTester/CloudToDeviceMessageReceiver.cs

```diff
@@ -46,14 +64,18 @@ public async Task ReportTestResult(Message message)
             await ModuleUtil.ReportTestResultAsync(this.testResultReportingClient, this.logger, testResultReceived);
         }
 
-        public override async Task StartAsync(CancellationToken ct)
+        public async Task StartAsync(CancellationToken ct)
         {
+            IEnumerable<X509Certificate2> certs = await CertificateHelper.GetTrustBundleFromEdgelet(new Uri(this.workloadUri), this.apiVersion, this.workloadClientApiVersion, this.moduleId, this.moduleGenerationId);
+            OsPlatform.Current.InstallCaCertificates(certs, ((Protocol)Enum.Parse(typeof(Protocol), this.transportType.ToString())).ToTransportSettings());
```

Comment:
Please put TODO here; AFAIK you can install certificate on Windows by script, you need to implement certificate verification callback handler.

Reply:
Sorry, my wrong comment.  "You cannot install certificate on Windows by script..."